src/UI/TransactionsToastList/components/TransactionToast/index.tsx

```diff
@@ -1,16 +1,14 @@
 import React from 'react';
-import classNames from 'classnames';
 import { Progress } from 'UI/Progress';
-import wrapperStyles from 'UI/TransactionsToastList/styles.scss';
+import { WithClassname } from '../../../types';
+import { useTransactionToast } from './hooks/useTransactionToast';
+import styles from './styles.scss';
```

Comment:
move styles below

Reply:
Can't. Eslint autoformatter is rearranging on save.